Compilers.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.25123.0
```

Comment:
@jaredpar What is this change?  I feel like I shouldn't be doing this, but VS did automatically.